cognite/client/data_classes/datapoints.py

```diff
@@ -171,12 +172,26 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
             dumped = {utils._auxiliary.to_camel_case(key): value for key, value in dumped.items()}
         return {key: value for key, value in dumped.items() if value is not None}
 
-    def to_pandas(self, column_names="externalId") -> "pandas.DataFrame":
+    def to_pandas(
+        self,
+        column_names: str = "externalId",
+        include_aggregate_name: bool = True,
+        complete: bool = None,
```

Comment:
This `complete` parameter should behave the same way as in retrieve_dataframe(). Or is there a good reason why it is a bool here?

Reply:
the dropna depends on all rows. This is a bit of an issue when retrieve can happily return a datapoints or datapointslist -- fixing now

Reply:
actually really annoying - any thoughts on how to 'always get a list' ?

Reply:
-> just making this a string as suggested, since the user may want to to_pandas it anyway